tools/integration_tests/emulator_tests/README.md

```diff
@@ -0,0 +1,8 @@
+# go-proxy-server
```

Comment:
IMO, there are two goals for this readme.
1. Provide steps to run/debug the existing tests.
2. Provide steps to add new tests in future.

We can take it in the end, for now we can add a TODO.